config/webpack/Plugins/ManifestTransformerPlugin.cjs

```diff
@@ -77,7 +84,7 @@ class ManifestTransformerPlugin {
 
                 results.push(match);
 
-                if (match.endsWith("*")) { continue; }
+                if (match.endsWith("*") || match.endsWith("/") || match.includes("?")) { continue; }
```

Comment:
`match.endsWith("/")` prevents adding malformed patterns when transforming `"*://store.steampowered.com/[?*]"` (the first content script entry).